sardine/src/tests/srd_tests.rs

```diff
@@ -116,6 +116,7 @@ fn good_login_logon_blob() {
     client_ciphers.push(Cipher::XChaCha20);
 
     let mut server_ciphers = Vec::new();
+    //server_ciphers.push(Cipher::AES256);
```

Comment:
Is this supposed to be commented?

Reply:
Yes! It was already commented for the client ciphers, so I left this one commented too so it's more consistent and easy to test. The reason both are commented is that you need to compile with the `aes` feature to enable it, which we don't tend to do in regular tests.